recipe/common.php

```diff
@@ -119,6 +119,7 @@
 option('tag', null, InputOption::VALUE_OPTIONAL, 'Tag to deploy');
 option('revision', null, InputOption::VALUE_OPTIONAL, 'Revision to deploy');
 option('branch', null, InputOption::VALUE_OPTIONAL, 'Branch to deploy');
+option('no-hooks', null, InputOption::VALUE_NONE, 'Run task without after/before hooks');
```

Comment:
@elfet Okay, I'll move it to TaskCommand. Meanwhile I think that it's okay to move all options to TaskCommand because of they are optional. Moreover, this way of defining options doesn't allow to add new ConsoleCommand with different arguments. I've already tried to add SSHCommand and faced that. I was not able to add required argument as there were defined options before. I guess that you will face it in version 5 refactoring.